package.json

```diff
@@ -89,6 +89,7 @@
     "moment-timezone": "^0.5.3",
     "ng-wig": "~2.3.4",
     "rx": "~4.0.8",
```

Comment:
I didn't do that yet in this PR. I was trying to be cautious about where it might conflict with changes in the Angular 2 migration